_dev/src/assets/scss/base/_common.scss

```diff
@@ -141,3 +141,28 @@ a.without-hover {
 .min-input-custom {
   max-width: 50px;
 }
+
+.circle-stepper-current {
+  border-radius: 50%;
+  min-width: 30px;
+  min-width: 30px;
+  background:  #D4E9D9;
+  color: #21834D;
+  line-height: 30px;
+  font-size: 17px;
+  text-align: center;
+  box-sizing: inherit!important;
+  &.circle-disabled {
+    background:  #D8D8D8;
+    color: #717171;
```

Comment:
Please define colors in:
* https://github.com/PrestaShopCorp/psxmarketingwithgoogle/blob/master/_dev/src/assets/scss/abstracts/_variables.scss
* https://github.com/PrestaShopCorp/psxmarketingwithgoogle/blob/master/_dev/src/assets/scss/base/_color.scss eventually